projects/step-core/src/lib/services/bookmark.service.ts

```diff
@@ -0,0 +1,69 @@
+import { inject, Inject, Injectable } from '@angular/core';
+import { BehaviorSubject, map, Observable, of } from 'rxjs';
+import { StorageProxy } from '../modules/basics/shared/storage-proxy';
+import { LOCAL_STORAGE } from '../modules/basics/shared/storage.token';
+import { Bookmark } from '../shared/bookmark';
+import { BookmarkStorageService } from './bookmark-storage.service';
+
+const bookmark = 'BOOKMARKS';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class BookmarkService extends StorageProxy {
```

Comment:
As storage has been moved to a separate service, this extension no longer required

Reply:
Extends removed